CHANGELOG.md

```diff
@@ -17,13 +17,21 @@ The REST API versions in the table are just for your information as the values a
 
 ## 7.0.0
 
+[Migration guide](https://sdk.gooddata.com/gooddata-ui/docs/migration_guide_7.html) from version 6.3.2
+
 ### Added
 
-- The source code structure is formatted and validated by [Prettier](https://prettier.io/) (FET-130)
+- Added Combo chart ([doc](https://sdk.gooddata.com/gooddata-ui/docs/combo_chart_component.html))
+- Pivot table subtotals can be managed by a submenu
+- The source code structure is formatted and validated by [Prettier](https://prettier.io/)
 
```

Comment:
- Combo chart component ([doc](https://sdk.gooddata.com/gooddata-ui/docs/combo_chart_component.html))
- Managing subtotals in pivot tables via a submenu
- Formatting and validating the source code structure by [Prettier](https://prettier.io/)